app/src/androidTest/java/org/mozilla/focus/activity/MozillaSupportPagesTest.kt

```diff
@@ -104,6 +105,7 @@ class MozillaSupportPagesTest {
 
     @SmokeTest
     @Test
+    @Ignore("Refactoring is required because of erase popup https://github.com/mozilla-mobile/focus-android/pull/6352")
```

Comment:
When disabling UI tests a new ticket should be creating to inform their owners about this and allow them to prioritize refactoring these tests.

Reply:
https://github.com/mozilla-mobile/focus-android/issues/6509